src/browser/telemetry.js

```diff
@@ -778,6 +792,51 @@ class Instrumenter {
     });
   }
 
+  /*
+   * Applies Rollbar telemetry scrubbing options to the dom input value.
+   * When replay options are present, applies those as well.
+   */
+  scrubInputValue(value, element, tagName, inputType) {
+    const mask = '******';
+
+    if (this.scrubTelemetryInputs) {
+      value = mask;
+    } else {
+      const description = domUtil.describeElement(element);
+      if (this.telemetryScrubber) {
+        if (this.telemetryScrubber(description)) {
+          value = mask;
+        }
+      } else if (this.defaultValueScrubber(description)) {
+        value = mask;
+      }
+    }
+
+    // Apply replay options regardless of other scrubbing
+    if (
+      this.maskInputOptions[tagName.toLowerCase()] ||
+      this.maskInputOptions[inputType]
+    ) {
+      if (this.maskInputFn) {
+        value = this.maskInputFn(value, element);
+      } else {
+        value = mask;
+      }
+    }
+
+    if (
+      domUtil.isMatchingElement(element, this.scrubClasses, this.scrubSelectors)
+    ) {
+      value = mask;
+    }
+
+    if (inputType === 'password') {
+      value = mask;
+    }
```

Comment:
Could this override the return value of the custom `this.maskInputFn`? Is that expected behavior?